tidepool-analysis-tools/tests/unit_tests/test_clean.py

```diff
@@ -0,0 +1,40 @@
+
+from tidals.clean.clean import remove_duplicates, round_time
+import pandas as pd
+from pandas.util import testing as tm
+import pytest
+from tests.fixtures.clean_setup import valid_df
+
+
+def test_remove_duplicates(valid_df):
+
+    duplicated_data =  [["jjety", "AB9960", "response1", "response2", "Sun Nov 19 23:59:59 2018", pd.to_datetime("2018-11-20 00:00:00")],
+                  ["jjety", "AB9958", "response1", "response2", "Sun Nov 19 23:56:59 2018", pd.to_datetime("2018-11-19 23:55:00")],
+                  ["hqoh", "AB9953", "response1", "response2", "Sun Nov 19 23:29:59 2018", pd.to_datetime("2018-11-19 23:30:00")],
+                  ["hhawe", "AB8769", "response1", "response2", "Sun Nov 19 23:20:01 2018", pd.to_datetime("2018-11-19 23:20:00")],
+                  ["hhawe", "AB8769", "response1", "response2", "Sun Nov 19 23:20:01 2018", pd.to_datetime("2018-11-19 23:20:00")]]
+
```

Comment:
only one space needed here. The convention we normally follow is one space between blocks of code and two spaces between sections (e.g., between definitions or major dividers).